### DISCUSSION TITLE
Guida per lo sviluppo di addons per Home Assistant

### DISCUSSION THREAD

**Discussion by officialdavide** [indomus/forum#501] [HUB: Home Assistant]
Ciao,
mi consigliate qualche guida mirata allo sviluppo di addons di Home Assistant?

Avrei necessità di sviluppare un addons che:
1) interroghi una fonte dati online, tramite API
2) riceva degli input da sensori zigbee
3) memorizzi alcuni dati in un database MySQL presente sul mio raspberry.

Grazie in anticipo!

**Comment by snakuzzo** [comment]
https://developers.home-assistant.io/docs/add-ons/
